### DISCUSSION TITLE
How to block some specific for all applications

### DISCUSSION THREAD

**Discussion by iLemonRain** [tnodir/fort#364] [Q&A] [answered]
Good day.
This applications works well, but how to block some specific traffic for all applications?
e.g. I'd like to block 10.192.33.35:80 for all apps
The feature seems done in menu OPTIONS->RULES->GLOBAL RULES
![img_v3_02hh_e9b21669-11bd-4ee8-9cdd-43842efb2f7g](https://github.com/user-attachments/assets/59fb44e6-527b-4099-bc31-faff9865a8d1)
So I tried with the content
![img_v3_02hh_2b77ff31-0bfe-4ab3-ac66-b0f494d2482g](https://github.com/user-attachments/assets/fb4701f5-1473-4e2f-8daf-1b6d35f6129d)
Unfortunately it not works. The traffic is not blocked. So please let me know how to do that or do you have the plan to add this feature?
Thanks a lot

**Comment by tnodir** [comment]
Hello.
Your Global Rule should work as you described.

How did you test it?

**Comment by tnodir** [comment, marked as answer]
Please check the [Filtering logic](https://github.com/tnodir/fort/wiki/Functionality-overview#filtering-logic):
```
1. If "Filter Enabled" is off, then PERMIT
...
6. If address is excluded from "Internet Addresses" and "Filter Local Network" is off, then PERMIT
```

So, the "10.192.33.35" address is defined as LAN address and not filtered in Fort by default, because Options: "Local Network Addresses" contains the "10.0.0.0/8".

You can turn on the Options: "Filter Local Network" flag to not auto-allow LAN addresses.

**Reply by iLemonRain** [reply]
That's the solution and thanks!
